src/Tizen.Applications.PackageManager/Tizen.Applications/Package.cs

```diff
@@ -288,6 +289,15 @@ public CertCompareResultType CompareCertInfo(string packageId)
             return (CertCompareResultType)compareResult;
         }
 
+        /// <summary>
+        /// The allowed packages and required privileges information.
+        /// </summary>
```

Comment:
It would be better to inform about the information that is contained in the dictionary. Even though it's rather clear.
```suggestion
        /// </summary>
        /// <remarks>
        /// The dictionary contains application IDs as the keys, and a collection of privileges related to that application as the value.
        /// </remarks>
```

Reply:
I added remarks.